build.sbt

```diff
@@ -135,17 +134,10 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .settings(commonWithPublishSettings)
   .settings(
     name               := "enumeratum",
-    version            := Versions.Core.head,
     crossScalaVersions := scalaVersionsAll,
-    libraryDependencies ++= {
-      if (useLocalVersion) {
```

Comment:
Interesting.

This was added as part of adding support for Scala 3.

https://github.com/lloydmeta/enumeratum/blame/c76f9487bc86b5fc744427e5450699d7f4558944/build.sbt#L150-L167

Does testing macro changes still work without this?

Reply:
All tests are passed both locally and on CI. Is there any preconditions to  verify this?

Reply:
I've just removed the rest of `configureWithLocal` and it looks like everything is works fine (I ran `+clean`, `+test`).